### PR TITLE
initial release of vcardgen library

### PR DESCRIPTION
**Pull request by vzaliva**

### CODE REVIEW

File: packages/vcardgen/vcardgen.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,12 @@
+opam-version: "1.2"
+maintainer: "lord@crocodile.org"
+authors: "Vadim Zaliva"
+homepage: "https://github.com/vzaliva/vcardgen"
+bug-reports: "https://github.com/vzaliva/vcardgen/issues"
+license: "BSD2"
+dev-repo: "https://github.com/vzaliva/vcardgen.git"
+build: ["jbuilder" "build"]
+install: ["jbuilder" "install"]
```
Comment (avsm) [inline packages/vcardgen/vcardgen.1.0/opam]: This `install` field is not needed, as jbuilder will generate an `install` file for you.

Comment (vzaliva) [reply packages/vcardgen/vcardgen.1.0/opam]: Thanks. I just removed it.

**Comment by vzaliva** [comment]
I pushed the change requested by @avsm in https://github.com/ocaml/opam-repository/pull/10365/commits/a0e1a441541c80d89ae86a58776288cdb4915684 but for some reason, it shows my commit before change request in GitHub pull request page.

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**